mitosheet/mitosheet/public/v3/formatting.py

```diff
@@ -8,6 +8,16 @@
 
 from mitosheet.excel_utils import get_column_from_column_index
 
+# Object to map the conditional formatting operators to the openpyxl operators
+operator_map = {
```

Comment:
In python, we usually make constants all caps -- so `OPERATOR_MAP`. Also, this could be more informatively named. Can you come up with a name that tells us specifically what's in the map?

Reply:
Ok! I added it as `CONDITIONAL_TO_OPENPYXL_OPERATOR_MAP`, is that too long?